frontend/src/views/AddPost/AddPost.js

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
 import { Container } from './AddPost.styled';
+import PostAddingForm from '../../components/PostAddingForm/PostAddingForm';
```

Comment:
```suggestion
import PostAddingForm from 'components/PostAddingForm/PostAddingForm';
```

tak jak Patryk zaznaczył, reszta super ;)